src/components/image.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from "react"
+
+interface Props {
+  image: string
+  className?: string
```

Comment:
A while ago we discovered that it is kind of an anti-pattern to pass class names to a component as it allows for a lot of variation. Respectively under certain circumstances, the appearance of the component can be completely different than what we intended in the components package. Instead of allowing to pass classes, we decided to identify the variations that the component should provide and define an interface for that. See for [example](https://carforyou.dev/carforyou-components-pkg/?path=/story/button--default) the `size` and `style` attribute on the button, that allows for a set of combinations but prevents the consumer from doing something like a yellow button with purple text. For the Image I would expect something like `object-fit` to be a potential variation. If you want to we can take a look together on Monday and define how the interface should look that we wanna provide for our images.

Reply:
Thank you for the feedback @lkappeler. I worked on an interface today, it is pretty straighforward to make it work for DH but more complicated for listings. I will try to fix my issues in listing tomorrow morning with a clear mind but I might ask for help indeed.